rust/lib/src/lib.rs

```diff
@@ -24,19 +24,7 @@ fn lock_client_return_seed(lightclient: LightClient) -> String {
 
     LIGHTCLIENT.lock().unwrap().replace(Some(lc));
 
-    let seed = match LIGHTCLIENT
-        .lock()
-        .unwrap()
-        .borrow()
-        .as_ref()
-        .unwrap()
-        .do_seed_phrase_sync()
-    {
-        Ok(s) => s.dump(),
-        Err(e) => {
-            return format!("Error: {}", e);
-        }
-    };
+    let seed = execute("seed".to_string(), "".to_string());
```

Comment:
in rust you can actually return this directly if you delete the closure ( ; )
i.e.
let seed = execute("seed".to_string(), "".to_string());
seed

is the same as:
execute("seed".to_string(), "".to_string())

Reply:
Good point... and thanks. I changed.